### ISSUE TITLE
Wii U USB HELPER Crashing

### ISSUE THREAD

**Issue by SMF9** [FailedShack/USBHelperLauncher#73]
Hello, I used to download games on Wii U Usb Helper but it suddently stopped working, it just crashes now and I got the logs.

```
-------------------------- Log Start --------------------------
Made by FailedShack
Updated translations: 60312-91954-5b436069e67aa
Injected module initializer.
Sending information to injector...
Wii U USB Helper returned non-zero exit code 0xe0434352:
Exception non gérée : System.Configuration.ConfigurationErrorsException: Échec de l'initialisation du système de configuration ---> System.Configuration.ConfigurationErrorsException: Fin de fichier inattendue. Les éléments suivants ne sont pas fermés : userSettings, configuration. Ligne 8, position 2876. (C:\Users\pixon\AppData\Local\Hikari06\WiiU_USB_Helper_.exe_Url_ymysly4xdeauh5vsedxqnhm2hqqlcbul\1.0.0.0\user.config line 8) ---> System.Xml.XmlException: Fin de fichier inattendue. Les éléments suivants ne sont pas fermés : userSettings, configuration. Ligne 8, position 2876.
   à System.Xml.XmlTextReaderImpl.Throw(Exception e)
   à System.Xml.XmlTextReaderImpl.ThrowUnclosedElements()
   à System.Xml.XmlTextReaderImpl.ParseElementContent()
   à System.Configuration.XmlUtil.StrictReadToNextElement(ExceptionAction action)
   à System.Configuration.BaseConfigurationRecord.ScanSectionsRecursive(XmlUtil xmlUtil, String parentConfigKey, Boolean inLocation, String locationSubPath, OverrideModeSetting overrideMode, Boolean skipInChildApps)
   à System.Configuration.BaseConfigurationRecord.ScanSectionsRecursive(XmlUtil xmlUtil, String parentConfigKey, Boolean inLocation, String locationSubPath, OverrideModeSetting overrideMode, Boolean skipInChildApps)
   à System.Configuration.BaseConfigurationRecord.ScanSections(XmlUtil xmlUtil)
   à System.Configuration.BaseConfigurationRecord.InitConfigFromFile()
   --- Fin de la trace de la pile d'exception interne ---
   à System.Configuration.ConfigurationSchemaErrors.ThrowIfErrors(Boolean ignoreLocal)
   à System.Configuration.BaseConfigurationRecord.ThrowIfParseErrors(ConfigurationSchemaErrors schemaErrors)
   à System.Configuration.ClientConfigurationSystem.OnConfigRemoved(Object sender, InternalConfigEventArgs e)
   --- Fin de la trace de la pile d'exception interne ---
   à System.Configuration.ClientConfigurationSystem.OnConfigRemoved(Object sender, InternalConfigEventArgs e)
   à System.Configuration.Internal.InternalConfigRoot.OnConfigRemoved(InternalConfigEventArgs e)
   à System.Configuration.Internal.InternalConfigRoot.RemoveConfigImpl(String configPath, BaseConfigurationRecord configRecord)
   à System.Configuration.BaseConfigurationRecord.GetSectionRecursive(String configKey, Boolean getLkg, Boolean checkPermission, Boolean getRuntimeObject, Boolean requestIsHere, Object& result, Object& resultRuntimeObject)
   à System.Configuration.BaseConfigurationRecord.GetSection(String configKey)
   à System.Configuration.ConfigurationManager.GetSection(String sectionName)
   à System.Configuration.ClientSettingsStore.ReadSettings(String sectionName, Boolean isUserScoped)
   à System.Configuration.LocalFileSettingsProvider.GetPropertyValues(SettingsContext context, SettingsPropertyCollection properties)
   à System.Configuration.SettingsBase.GetPropertiesFromProvider(SettingsProvider provider)
   à System.Configuration.SettingsBase.GetPropertyValueByName(String propertyName)
   à System.Configuration.SettingsBase.get_Item(String propertyName)
   à System.Configuration.ApplicationSettingsBase.GetPropertyValue(String propertyName)
   à System.Configuration.ApplicationSettingsBase.get_Item(String propertyName)
   à WIIU_Downloader.Properties.Settings.get_AppId()
   à A.c4a33c9ca97b5b0cfc40918375bc969d1.ce0be635e443d424596522127e6756824(String[] c3c9260886a6db15de49329a9f5b9836c)
---------------------- Fiddler Log Start ----------------------
Fiddler ICertificateProvider v5.0.0.0 loaded.
	fiddler.certmaker.bc.Debug:	False
	ObjectID:			0x245fb7
Setting upstream gateway to none
```

**Comment by DaKux** [comment]
So yeah if anyone have an idea on what is happening well

**Comment by eltmanm** [comment]
Encountering same problem after trying to uninstall and reinstall program as much from scratch as possible:
------------- USBHelperLauncher Debug Information -------------
Debug Time: 1/20/2021 4:42:46 AM (UTC)
Session Length: 00:00:10
Session GUID: dfdec803-0d05-43df-8fb6-538070a4c467
Proxy Available: Yes
Public Key Override: Yes
Version: 0.17d
Helper Version: 0.6.1.653
.NET Framework Version: 4.8 or later
Operating System: Microsoft Windows 10 Pro (64-bit)
Platform: Win32NT
Used Locale: en-US
System Language: en-US
Total Memory: 16074747904
Available Memory: 9976934400
Antivirus Software:
Windows Defender -> Enabled
Endpoint Fallbacks:
ContentEndpoint -> https://cdn.shiftinv.cc/wiiuusbhelper/cdn/
-------------------------- Log Start --------------------------
Made by FailedShack
Updated translations: 60312-91e79-5b94d69c42c2c
Injected module initializer.
Sending information to injector...
Wii U USB Helper returned non-zero exit code 0xe0434352:
Unhandled Exception: System.TypeInitializationException: The type initializer for '<Module>' threw an exception. ---> HarmonyLib.HarmonyException: Patching exception in method static System.Reflection.MethodBase USBHelperInjector.Patches.MoviePlaybackPatch::TargetMethod() ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.IO.FileNotFoundException: Could not load file or assembly 'SharpDX.DXGI, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b4dcf0f35e5521f1' or one of its dependencies. The system cannot find the file specified.
   at System.Signature.GetSignature(Void* pCorSig, Int32 cCorSig, RuntimeFieldHandleInternal fieldHandle, IRuntimeMethodInfo methodHandle, RuntimeType declaringType)
   at System.Signature..ctor(IRuntimeFieldInfo fieldHandle, RuntimeType declaringType)
   at System.Reflection.RtFieldInfo.get_FieldType()
   at USBHelperInjector.Patches.MoviePlaybackPatch.<>c.<TargetMethod>b__7_2(<>f__AnonymousType4`2 <>h__TransparentIdentifier0)
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at USBHelperInjector.Patches.MoviePlaybackPatch.TargetMethod()
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at HarmonyLib.PatchClassProcessor.RunMethod[S,T](T defaultIfNotExisting, T defaultIfFailing, Func`2 failOnResult, Object[] parameters)
   --- End of inner exception stack trace ---
   at HarmonyLib.PatchClassProcessor.ReportException(Exception exception, MethodBase original)
   at HarmonyLib.PatchClassProcessor.Patch()
   at USBHelperInjector.InjectorService.<>c.<Init>b__40_1(Type type)
   at HarmonyLib.CollectionExtensions.Do[T](IEnumerable`1 sequence, Action`1 action)
   at USBHelperInjector.InjectorService.Init()
   at .cctor()
   --- End of inner exception stack trace ---
---------------------- Fiddler Log Start ----------------------
Fiddler ICertificateProvider v5.0.0.0 loaded.
	fiddler.certmaker.bc.Debug:	False
	ObjectID:			0x2a125d8
Setting upstream gateway to none

**Comment by OZtistic** [comment]
> HarmonyLib.HarmonyException: Patching exception in method static System.Reflection.MethodBase
you have installed it wrong. either you did half the setup of the manual install, or you did not use the auto installer (which I would recommend.)
https://github.com/FailedShack/USBHelperInstaller/releases/download/1.2/USBHelperInstaller.exe
please close this issue if you have no further issues @SMF9 @eltmanm
